css/style.css

```diff
@@ -1525,6 +1547,242 @@ input:focus-visible {
   line-height: 1.33;
 }
 
+.single-article-meta {
+  display: grid;
+  grid-template-columns: 1fr 150px 150px;
+  align-items: center;
+}
+
+.single .about-author {
+  display: flex;
+  align-items: center;
+}
+
+.single .author-avatar {
+  border-radius: 50%;
+  overflow: hidden;
+  width: 48px;
+  height: 48px;
+  margin-right: 13px;
+}
+
+.single .meta-author-circle {
+  width: 48px;
+  height: 48px;
+  background: rgba(15, 54, 46, 0.08);
+  display: flex;
+  border-radius: 50%;
+}
+
+.single .author-info a {
+  display: block;
+  color: var(--dark);
+}
+
+.single .author-info span {
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 600;
+  line-height: 16px;
+}
+
+.single .meta-comments-count, 
+.single .social {
+  background: #CBEFB6;
+  height: 32px;
+  padding: 4px 16px;
+  justify-content: center;
+  align-items: center;
+  border-radius: 80px;
+}
+
+.single .article-cover-image {
+  margin: 24px 0 80px 0;
+}
+
+.single .article-cover-image > div {
+  min-height: 480px;
+  border-radius: var(--rounded);
+}
+
+.single .row > #main {
+  padding: 0 10%;
+}
+
+.single .bulletpoints {
+  margin-bottom: 80px;
+}
+
+.toggle-bulletpoints {
+  width: 100%;
+  height: 100px;
+  margin-top: -90px;
+  background: linear-gradient(to bottom, transparent, #fff, #fff);
+  position: absolute;
+  left: 0;
+  border: none;
+}
+
+.dark-skin .toggle-bulletpoints {
+  background: linear-gradient(to bottom, transparent, #000, #000);
+}
+
+.toggle-bulletpoints span {
+  line-height: 150px;
+  color: var(--brand-color);
+  font-size: 14px;
+  font-weight: 600;
+}
+
+.toggle-bulletpoints span:after {
+  content: "\e80c";
```

Comment:
Most likely custom font needs an update as there is no glyph with such code yet. I will update custom font but then the code should match.

Reply:
@martinosmartinos Is it possible to retrieve the fonts we have in jannah/img/font, where we have all the necessary glyphs?

Reply:
@Right-Noe sure, within this PR please feel free to update font files, fontello.css and all the references to glyphs in style.css.